tiledb/cloud/results.py

```diff
@@ -127,3 +132,20 @@ def _maybe_uuid(id_str: Optional[str]) -> Optional[uuid.UUID]:
         return uuid.UUID(hex=id_str)
     except ValueError:
         return None
+
+
+# Compatibility aliases.
+# Until a version with decoders.py is deployed to the UDF environment,
```

Comment:
We might need to maintain these aliases for longer due to stored UDFs...

Reply:
These never need to be handled directly by user code, so they should not need to stay around. I have a plan to seamlessly remove these before officially releasing this feature that I will detail more on internal channels.